apps/studio/src/components/tableview/TableTable.vue

```diff
@@ -647,6 +647,11 @@ export default Vue.extend({
             verticalNavigation: useVerticalNavigation ? 'editor' : undefined,
             search: true,
             allowEmpty: true,
+            preserveObject: column.dataType.startsWith('_'),
+            onPreserveObjectFail: () => {
```

Comment:
Probably this should just log to the console or something?

If there are 500 rows, this will trigger 500 times

Reply:
Oh yeah, that's really bad 😆